designTokens/size.ts

```diff
@@ -81,6 +81,40 @@ export const size: DesignTokens = {
   },
   form: {
     element: {
+      s: {
```

Comment:
this is probably xs?
<img width="1152" alt="Bildschirmfoto 2024-04-04 um 13 28 33" src="https://github.com/lyne-design-system/lyne-design-tokens/assets/29278232/d0c99c03-0943-47b3-9aac-1d54620bfbb9">

Reply:
i named it 's' as in the text-input in Figma.. 
how could i reach the page in the screenshot? i see nothing here https://lyne.sbb.ch/tokens/size/